stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -143,15 +143,13 @@ private ClientCalls() {}
    * <p>If interrupted, the interrupt is restored before throwing a {@code RuntimeException}.
    *
    * @throws RuntimeException if {@code get} is interrupted
-   * @throws CancellationException if {@code get} throws a {@code CancellationException}
-   * @throws StatusRuntimeException if {@code get} throws an {@code ExecutionException}
```

Comment:
Remove the throws RuntimeException line and add "or interrupted" to this line, since we throw StatusRuntimeException in both cases?